plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -56,6 +56,10 @@
   @Override
   protected Control createContents(Composite parent) {
     Composite container = new Composite(parent, SWT.NONE);
+    
+    PlatformUI.getWorkbench().getHelpSystem().setHelp(parent,
```

Comment:
Setting this on `parent` bothers me a bit, but I couldn't find an alternative. :-/ (If set on `container`, the help shows only when the controls have focus.)